book/installation/binaries.md

```diff
@@ -1,28 +1,18 @@
 # Binaries
 
-Precompiled binaries are available from the [GitHub releases page](https://github.com/paradigmxyz/reth/releases).
+[**Archives of precompiled binaries of reth are available for Windows, macOS and Linux.**](https://github.com/paradigmxyz/reth/releases) They are static executables. Users of platforms not explicitly listed below should download one of these archives.
 
-Binaries are supplied for 5 platforms:
+If you use **macOS Homebrew** or **Linuxbrew**, you can install Reth from Paradigm's homebrew tap:
 
-- Linux x86_64: AMD and Intel 64-bit processors (most desktops, laptops, and servers)
-- Linux ARM64: 64-bit arm processors
-- macOS x86_64: macOS with Intel chips
-- macOS ARM64: macOS with Apple Silicon
-- Windows x86_64: AMD and Intel 64-bit processors
+```text
+brew install paradigmxyz/brew/reth
+```
 
-Each binary is contained in a tarball.
+If you use **Arch Linux** you can install stable Reth from the AUR using an [AUR helper](https://wiki.archlinux.org/title/AUR_helpers) ([paru][paru] as an example here):
 
-As an example, you could install the Linux x86_64 version like so:
+```text
+paru -S reth # Stable
```

Comment:
Does this also work with `pacman -S reth`?

Reply:
No, pacman only installs packages maintained in the core and extra repos, and only popular packages can get in there. But most Arch users have an AUR helper installed since a lot of cool software lives there